ethcore/src/evm/evm.rs

```diff
@@ -79,6 +82,12 @@ impl From<builtin::Error> for Error {
 	}	
 }
 
+impl From<wasm::RuntimeError> for Error {
+	fn from(_err: wasm::RuntimeError) -> Self {
+		Error::Wasm("Runtime error")
```

Comment:
I see, but there are also a lot of wasm errors with allocated strings that will be discarded here. Perhaps an inner `enum WasmError { Runtime, Other(String) }` so that extra info isn't lost?